.github/workflows/code-quality.yaml

```diff
@@ -112,3 +112,8 @@ jobs:
               git diff
               exit 1
           fi
+      # This simply checks that the manifests and respective kustomization.yaml finishes without an error.
+      - name: Check kustomize manifest
+        id: kustomize-manifests
+        run: |
+          ./ci/kustomize.sh
```

Comment:
Thank you for this addition. The code itself seems good. But I wouldn't add it as a step into this existing `check-generated-code` job. Looking on the jobs in this file, I think we should introduce a new job - let's inspire by the [notebooks repository](https://github.com/opendatahub-io/notebooks/blob/6e4cf510f7e9c1390c0871d878fb1e52d3b1060f/.github/workflows/code-quality.yaml#L60-L63) again - so something like:
```
  code-static-analysis:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4

      # This simply checks that the manifests and respective kustomization.yaml finishes without an error.
      - name: Check kustomize manifest
        id: kustomize-manifests
        run: |
          ./ci/kustomize.sh
```